sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1687,14 +1679,6 @@ object SQLConf {
       "the SQL parser.")
     .fallbackConf(ANSI_ENABLED)
 
-  val ALLOW_CREATING_MANAGED_TABLE_USING_NONEMPTY_LOCATION =
```

Comment:
Was this explicitly to be removed in 3.0? doesn't say so in the doc but it may have been otherwise documented or well understood.

Reply:
Yes, it is mentioned in the SQL migration guide: https://github.com/apache/spark/blob/80fbc382a60973db21367a922a0fb797e5ab382d/docs/sql-migration-guide.md#L315